test/end-to-end.js

```diff
@@ -129,23 +129,6 @@ describe('end-to-end', function() {
       // the test will time out if `booted` is not emitted
     });
 
-    it('has legacy explorer disabled in config', function(done) {
-      expect(app.get('legacyExplorer'), 'legacyExplorer option').to.be.false();
-      done();
-    });
-
-    it('has legacy explorer route /models disabled', function(done) {
-      request(app)
-        .get('/api/models')
-        .expect(404, done);
-    });
-
-    it('has legacy explorer route /routes disabled', function(done) {
-      request(app)
-        .get('/api/routes')
-        .expect(404, done);
-    });
-
```

Comment:
I think it's better to keep these tests around. For 2.x apps, the routes should be disabled by config. For 3.x+ apps, the routes should not be there because they are not implemented at all. In all cases, these tests should pass.